### DISCUSSION TITLE
Buttons of the Home Page

### DISCUSSION THREAD

**Discussion by jczanfona** [aleksey-hoffman/sigma-file-manager#122] [Q&A]
Why the buttons of the Home Page hasn't the button behavior and is necessary to click twice?

With one one click does nothing, without double click does nothing...  if they are buttons they should be activated with a single click.

**Comment by aleksey-hoffman** [comment]
The directories on the home page behave similarly to the directories on the navigator page.
You can change this behavior in settings.

![image](https://user-images.githubusercontent.com/61761672/124933604-b1abfc80-e00c-11eb-9db4-a8475dcfa4c2.png)

**Reply by jczanfona** [reply]
ok, that can be a solutions (everything like button)
